source/stream.js

```diff
@@ -1,7 +1,7 @@
 import {isReadableStream} from 'is-stream';
 
 export const getAsyncIterable = stream => {
-	if (isReadableStream(stream, {checkOpen: false})) {
+	if (isReadableStream(stream, {checkOpen: false}) && nodeImports.on !== undefined) {
```

Comment:
In principle, browsers do not support Node streams, so if `stream` is a Node `Readable`, then `nodeImports.on` should be defined.

However, there might be some exotic environment which would load the browser entrypoint _and_ have Node streams support. For example, maybe Deno.